### PR TITLE
[4.1] Codemirror Full Screen

### PR DESCRIPTION
**Pull request by brianteeman**
This PR re-introduces the on screen instruction how to make codemirror full screen. (No idea why it was removed the pr doesnt help) AND fixes the positioning of the editor when full screen so that the first 5 lines are not hidden.

As there is a css change you will need to use a pre-build package or `npm run build:css`

Easiest to test in com_templates as that uses codemirror by default

### CODE REVIEW

**Comment by Quy** [comment]
How about float right to be subtle? 

37129-codemirror

**Comment by brianteeman** [comment]
updated as requested

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php [hunk 1]
```diff
@@ -55,5 +56,8 @@
 ?>
 <joomla-editor-codemirror <?php echo $editor . ' ' . $addons . ' ' . $modPath . ' ' . $fsCombo . ' ' . $option; ?>>
 <?php echo '<textarea name="', $name, '" id="', $id, '" cols="', $cols, '" rows="', $rows, '">', $content, '</textarea>'; ?>
+<p class="small float-end">
+	<?php echo Text::sprintf('PLG_CODEMIRROR_TOGGLE_FULL_SCREEN', $modifier, $params->get('fullScreen', 'F10')); ?>
```
Comment (Quy) [inline plugins/editors/codemirror/layouts/editors/codemirror/element.php]:
```suggestion
	<?php echo Text::sprintf('PLG_CODEMIRROR_TOGGLE_FULL_SCREEN', $fullScreenCombo); ?>
```

File: administrator/language/en-GB/plg_editors_codemirror.ini [hunk 2]
```diff
@@ -39,5 +39,6 @@ PLG_CODEMIRROR_FIELD_VALUE_SCROLLBARSTYLE_SIMPLE="Simple"
 PLG_CODEMIRROR_FIELD_VIM_KEYBINDING_LABEL="Vim Keybinding"
 PLG_CODEMIRROR_FIELDSET_APPEARANCE_OPTIONS_LABEL="Appearance Options"
 PLG_CODEMIRROR_FIELDSET_TOOLBAR_OPTIONS_LABEL="Toolbar Options"
+PLG_CODEMIRROR_TOGGLE_FULL_SCREEN="Press %1$s %2$s to toggle Full Screen editing."
```
Comment (Quy) [inline administrator/language/en-GB/plg_editors_codemirror.ini]:
```suggestion
PLG_CODEMIRROR_TOGGLE_FULL_SCREEN="Press %s to toggle Full Screen editing."
```

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php (hunk 1 above)
Comment (brianteeman) [reply plugins/editors/codemirror/layouts/editors/codemirror/element.php]: this is not correct. There is a modifier as well as a key

File: administrator/language/en-GB/plg_editors_codemirror.ini (hunk 2 above)
Comment (brianteeman) [reply administrator/language/en-GB/plg_editors_codemirror.ini]: this is not correct. There is a modifier as well as a key

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php (hunk 1 above)
Comment (Quy) [reply plugins/editors/codemirror/layouts/editors/codemirror/element.php]: It is correct. Please see lines 30-32.

Comment (brianteeman) [reply plugins/editors/codemirror/layouts/editors/codemirror/element.php]: my mistake - never spotted that. maybe its also wrong in j3?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 95fd2bbae0244a40ec9e8ccf84d91872e518e6e5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37129">issues.joomla.org/tracker/joomla-cms/37129</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 95fd2bbae0244a40ec9e8ccf84d91872e518e6e5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37129">issues.joomla.org/tracker/joomla-cms/37129</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37129">issues.joomla.org/tracker/joomla-cms/37129</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**